templates/microcloud/index.html

```diff
@@ -0,0 +1,331 @@
+{% extends "templates/base.html" %}
+
+{% block title %}MicroCloud — 打造开源、灵活、可扩展的云端架构 {{ page_title }}{% endblock %}
```

Comment:
For some reason the meta title isn't being set properly
![image](https://github.com/user-attachments/assets/4b3ed357-352d-4257-ad56-cb15b39f7870)